CMakeLists.txt

```diff
@@ -36,8 +36,7 @@ if (BUILD_DEPS)
 
     add_subdirectory(crt/aws-c-common)
     if (UNIX AND NOT APPLE)
-        list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/crt/s2n/cmake/modules")
-        find_package(LibCrypto REQUIRED)
+        include(crt/aws-c-cal/cmake/modules/aws-lc.cmake)
```

Comment:
> 2021-03-18T21:46:07.2383345Z -- Installing: /usr/local/lib/libcrypto.a

Looking at the logs, aws-lc is getting installed to a permanent location instead of some local build directory

I think this is because:
- line 39 includes aws-lc.cmake, which builds and installs it as a side effect
- line 60 manually sets `CMAKE_INSTALL_PREFIX` to the local build directory

probably want to move line 60 up